app/models/ontology_version/proving.rb

```diff
@@ -0,0 +1,38 @@
+module OntologyVersion::Proving
+
+  extend ActiveSupport::Concern
+  include Hets::ErrorHandling
+
+  included do
+    @queue = 'hets'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.